src/fa/game_session.py

```diff
@@ -177,13 +178,41 @@ def _turn_state_changed(self, val):
 
     def _launched(self):
         self._logger.info("Game has started")
+        with open(util.LOG_FILE_GAME, 'r') as src, open(util.LOG_FILE_GAME+'.full.log', 'a+') as dst: dst.write(src.read())
 
     def _exited(self, status):
         self._game_connection = None
         self.state = GameSessionState.OFF
         self._logger.info("Game has exited with status code: {}".format(status))
         self.send('GameState', ['Ended'])
 
+        replayID=''
+        with open(util.LOG_FILE_GAME,'r') as inF:
+            for line in inF:
+                if 'gpgnet://localhost/' in line:
+                    try:
+                        replayID=line.split('localhost/')[1].split('/')[0]
```

Comment:
The class already has a game_uid member that gets set at game launch. Let's use it instead of hacking things.

Reply:
Oh indeed, didn't notice there was game_uid there already made. Removed this hacky solution